collectors/statsd.plugin/README.md

```diff
@@ -1,10 +1,8 @@
-<!--
+---
 title: "statsd.plugin"
 description: "The Netdata Agent is a fully-featured StatsD server that collects metrics from any custom application and visualizes them in real-time."
 custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/statsd.plugin/README.md
```

Comment:
Why changing `<!--`?

I see this syntax is being used in all markdown files.

```cmd
netdata (master) $ grep --include=*.md "<\!--" -R * | wc -l
341
```

Reply:
@ilyam8 
I wasn't aware that we use a script that greps Markdown files like this. 
I proposed the change because usually, the standard way to include frontmatter is using three dashes, see[Docusaurus documentation](https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-docs#markdown-frontmatter) (After the table).

Reply:
> we use a script that greps Markdown files like this

Hm, not sure why do you mean, I just did a quick check and count the number of markdown files that start with `<!--`.

I noticed there is a difference between `<!--` and `---` - the former is not visible on the page. Compare [before](https://github.com/netdata/netdata/blob/master/collectors/statsd.plugin/README.md) vs [after](https://github.com/netdata/netdata/blob/3ba594892d68e197453aabd15fff67c10a0f80a7/collectors/statsd.plugin/README.md). Is it expected to have this table on the top?

Reply:
@ilyam8 
Ah okay, so the grep command above is not used anywhere in production? 

If you read the documentation directly in github, then yes, it's expected that the metadata from the front matter are rendered as a table. I do understand that it might distract people, though. 

So, as long as `<!-- -->` and `--- ---`do the same thing in Docusaurus, I am willing to keep the non-conformant way to implement front matter. 
However, since this is normally syntax for a HTML comment, I wasn't 100% sure if the metadata would be usable. @DShreve2 Do you know more about this?

Reply:
> Ah okay, so the grep command above is not used anywhere in production?

Yes. It is just me executing the command to show you the number of occurrences of files with `<!---`.

Reply:
@ilyam8 So it looks like that our documentation setup handles `<!-- -->` and `--- ---` the same way. I'm fine to keep the `<!-- -->` syntax. 
Thanks for the question, it helped me understand our systems better. 👍